### ISSUE TITLE
Crash when tapping on an update notification

### ISSUE THREAD

**Issue by opusforlife2** [NeoApplications/Neo-Store#526]
I had Neo Store (v0.9.15) previously opened in the background. After that I used several other apps, which presumably kicked Neo Store out of memory. Then I tapped on the update notification, and Scoop caught this error.

```
FATAL EXCEPTION: main
Process: com.machiav3lli.fdroid, PID: 25426
java.lang.RuntimeException: Unable to resume activity {com.machiav3lli.fdroid/com.machiav3lli.fdroid.ui.activities.MainActivityX}: kotlin.UninitializedPropertyAccessException: lateinit property navController has not been initialized
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4824)
	at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:4857)
	at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:54)
	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2253)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7870)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
Caused by: kotlin.UninitializedPropertyAccessException: lateinit property navController has not been initialized
	at com.machiav3lli.fdroid.ui.activities.MainActivityX.handleSpecialIntent(MainActivityX.kt:21)
	at com.machiav3lli.fdroid.ui.activities.MainActivityX.handleIntent(MainActivityX.kt:73)
	at com.machiav3lli.fdroid.ui.activities.MainActivityX.onNewIntent(MainActivityX.kt:4)
	at android.app.Activity.performNewIntent(Activity.java:8073)
	at android.app.Instrumentation.callActivityOnNewIntent(Instrumentation.java:1441)
	at android.app.Instrumentation.callActivityOnNewIntent(Instrumentation.java:1454)
	at android.app.ActivityThread.deliverNewIntents(ActivityThread.java:3904)
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4807)
	... 13 more
```

**Comment by machiav3lli** [comment]
Same as the other issue applies, please report if this's reproducible on 1.0.0-alpha06.

**Comment by machiav3lli** [comment]
Thanks, I'll see into it

**Comment by machiav3lli** [comment]
Should be fixed in 1.0.0

**Comment by xa4hf8** [comment]
The issue persists in 1.0.9
